### ISSUE TITLE
Cannot use the wallet

### ISSUE THREAD

**Issue by Sovxx** [deroproject/derohe#26]
Hi,

Config :
- DERO-HE STARGATE Mainnet Release45 (dero_windows_amd64.zip)
- Windows 10
- GIT bash

Daemon looks fine :
![daemon](https://user-images.githubusercontent.com/12556923/155877717-b04e1312-e29f-4c40-a094-93f4891b802f.png)

But when I open the wallet, it looks like this (messy) :
![wallet](https://user-images.githubusercontent.com/12556923/155877724-07494eca-7706-49a9-b25a-89ab7540696b.png)

Also, I cannot type more than one character in the prompt.
And when I type "2" + Enter, I just get another prompt ; it does not seem to create a new wallet.

Thx

**Comment by Slixe** [comment]
Can you try using windows CMD directly ?

**Comment by Sovxx** [comment]
It works !

with cmd
![cmd](https://user-images.githubusercontent.com/12556923/155882323-917964f8-f403-4917-9d88-d233a25bed81.png)

with powershell
![powershell](https://user-images.githubusercontent.com/12556923/155882331-ee4f3116-e387-4618-88cb-295b263fd2b3.png)

Thanks

**Comment by triplefox** [comment]
I am encountering similar failures to this issue when running from Windows cmd. As of Release 50 of cli wallet I experience these issues:

* Flickering status line display and mixed input and output similar to the screenshots in the original posting
* Drops the input of characters "b" and "d", even when copy-pasting.
* Backspace deletes multiple characters or entire line

At first I only saw this behavior occasionally at mainnet launch, but it has now become consistent, rendering the wallet impossible to use.

Some speculation: These behaviors, particularly the strange dropped/duplicated input, are consistent in presentation with threaded concurrency bugs I've encountered in other Windows console apps, and a rise in network activity could explain why it is now more frequent. It may also be the case that it's triggered by certain machines, core counts and environments(the machine experiencing it is a weaker Pentium N5000 laptop with 4c/4t), or are triggered by a Windows-specific interaction or thread scheduling method. These would account for it working in cmd on some machines and not others, or for it failing to show in VM testing. When I run the wallet on my Linux machine, they do not present.
